c/parallel/src/jit_templates/traits.h

```diff
@@ -0,0 +1,54 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Experimental in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <cuda/std/type_traits>
+
+#include <format>
+#include <string>
+
+#include "../util/errors.h"
+
+extern const char* jit_template_header_contents;
+
+template <typename Arg>
+struct parameter_mapping;
+
+template <typename Tpl>
+struct template_id
+{};
+
+struct specialization
```

Comment:
Could this `struct` be renamed to `specialization_t` or `specialization_st` to make it easier to realize that it describes a type. Ease mental burden of parsing the code.